containers/chrome-headless/Dockerfile

```diff
@@ -9,11 +9,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.16.1
-
-RUN curl -sSL https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - \
-    && curl -sL https://deb.nodesource.com/setup_14.x | bash - \
-    && apt install nodejs
+FROM quay.io/kubermatic/nodejs:v2.0
```

Comment:
`quay.io/kubermatic/nodejs` is pretty much identical to golang+nodejs. Just a renaming.